scripts/fast-reboot

```diff
@@ -92,7 +92,7 @@ function initialize_pre_shutdown()
 function request_pre_shutdown()
 {
     debug "Requesting pre-shutdown ..."
-    /usr/bin/docker exec -i syncd /usr/bin/syncd_request_shutdown --pre
+    /usr/bin/docker exec -i syncd /usr/bin/syncd_request_shutdown --pre &> /dev/null
```

Comment:
Are we sure that this task will be finished before we requested system reboot (or kill all docker containers)?

Reply:
Never knew about this syntax. Thank you for heads up